db_schema/queries/v1/reportGenericHosterBaseMetrics.sql

```diff
@@ -0,0 +1,32 @@
+-- Returns both total metrics and metrics for a given period (e.g., last 30 days)
+-- Only sums daily metrics (start = end)
+-- Covers downloads, platforms, clients, sources
+
+SELECT
+    account_id AS podcast_id,
+    hoster_id,
+    dimension,
+    subdimension,
+    SUM(CASE 
+            WHEN start >= @start_date AND start < @end_date AND start = end
+            THEN value 
+            ELSE 0 
+        END) AS value_period,
+    SUM(CASE 
```

Comment:
as we do not have daily values for all dates since publishing this doesn not return the lifetime total.

Reply:
done in 70a0664c346a80c148aeb35e6e140aad6c3c7283